stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1721,6 +1725,18 @@ extension ${Self} {
 }
 
 % if bits == 80:
+#else
+${SelfDocComment}
```

Comment:
Please fix-up the indentation here. Also, @moiseev, do we want `@_transparent`, etc on a dummy object like this?

Reply:
I just copied the main definition, added the `@available` and removed everything referencing Builtin.FPIEEE